### PR TITLE
Event model

### PR DESCRIPTION
**Pull request by Woulei**
Did not make the associations with the workshop and sponsor yet. Rails won't allow me because they don't exist yet. Need to do that later. Same for the image.

### CODE REVIEW

File: spec/models/event_spec.rb [hunk 1]
```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+RSpec.describe Event, type: :model do
+  describe "Check validations" do
```
Comment (arnoFleming) [inline spec/models/event_spec.rb]: you could just name it like so: 
```ruby
describe "validations" do
  context "title" do
    # specs
  end
end
```

File: spec/models/event_spec.rb [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      it "is invalid with an decimal" do
+        event = Event.new(spaces: 0.5)
+        event.valid?
+        expect(event.errors).to have_key(:spaces)
+      end
+      it "is valid with an integer" do
+        event = Event.new(spaces: 5)
+        event.valid?
+        expect(event.errors).not_to have_key(:spaces)
+      end
```
Comment (arnoFleming) [inline spec/models/event_spec.rb]: You could have a couple of specs exposing all invalid cases, and then have **one spec** for valid data

``` ruby
it "is valid with valid attributes" do
  event  = Event.new(
    title: "An new title",
    location: "Lee's place",
    spaces: 25, 
  )
  expect(event.valid?).to eq(true)
end
```

**Comment by Woulei** [comment]
@arnoFleming How can we resolve the schema.rb conflict? Because we all will have different version numbers?

**Comment by arnoFleming** [comment]
re: schema conflicts: In your local repo, 

- checkout master, 
- git pull, 
- checkout your branch, 
- merge master into it
- run a db:migrate
- add the schema file 

and you're done

**Pull request merged**